### DISCUSSION TITLE
Having problems with filebrowser as service and after upload hook.

### DISCUSSION THREAD

**Discussion by Russsgithub** [filebrowser/filebrowser#2110] [Q&A]
I've been using the after upload hook to add metadata to mp3s , which works fine if filebrowser is run from a terminal.
If I run it as a service

```
[Unit]
Description=Filebrowser
After=network-online.target

[Service]
User=root
Group=root
ExecStart=/usr/local/bin/filebrowser  -r /home/ubuntu/files

[Install]
WantedBy=multi.user.target
```

The files are created with root ownership and when they are rewritten it fails on permissions error.

I've tried using a non root user in the service file to get around this, but get ```failed to open filebrowser.db``` when the service starts even if i delete the db file beforehand.

Any advice would be appreciated.
Thanks

I see there is a discussion related to a similar issue using the docker file.

**Comment by ramiresviana** [comment]
Running Filebrowser with no configuration will cause it to create a database file on the current working directory. If you want to utilize an user other than root, you must provide a writable directory.
